plugins/inputs/aerospike/aerospike.go

```diff
@@ -436,13 +455,78 @@ func parseAerospikeValue(key string, v string) interface{} {
 		return parsed
 	} else if parsed, err := strconv.ParseBool(v); err == nil {
 		return parsed
+	} else if parsed, err := strconv.ParseFloat(v, 64); err == nil {
+		return parsed
 	} else if parsed, err := strconv.ParseFloat(v, 32); err == nil {
 		return parsed
```

Comment:
Is this the right behavior? In what cases would float32 still be parsed here, and won't this change metric field types?

Reply:
Removed the float64 parse logic.

Reply:
I've changed the float parsing to 64 bits, which will return two digits(40.98), and 32 bits will return 14 digits(40.97999954223633).